NYPLAudiobookToolkit/UI/AudiobookPlayerViewController.swift

```diff
@@ -238,25 +285,29 @@ public final class AudiobookPlayerViewController: UIViewController {
         func actionFrom(trigger: SleepTimerTriggerAt, sleepTimer: SleepTimer) -> UIAlertAction {
             let handler = { (_ action: UIAlertAction) -> Void in
                 sleepTimer.setTimerTo(trigger: trigger)
+                self.sleepTimerValueShouldUpdate()
```

Comment:
non-blocking: I know I named these methods originally, but they really should be named like this:

```swift
func updateSleepTimerIfNeeded()
func updateSpeedButtonIfNeeded(rate: PlaybackRate)
```

the suffix `ShouldUpdate` implies that they will return a boolean and is semantically awkward. `IfNeeded` follows method names in `UIView` and properly indicates they will perform an action if a requirement is met.

Reply:
we also use the `IfNeeded` in many places, so these should totally be changed.

Reply:
agreement: Sounds good!